src/lib.rs

```diff
@@ -265,6 +265,37 @@ impl Ulid {
         }
     }
 
+    /// Increment the random number, increasing the timestamp if the random number overflows
+    ///
+    /// # Panics
```

Comment:
Please add a test for the panic case. `#[should_fail]` can be used mark a unit test that is meant to fail.

Reply:
Done :)